core/field_angle.js

```diff
@@ -111,11 +131,14 @@ Blockly.FieldAngle.prototype.dispose_ = function() {
   return function() {
     Blockly.FieldAngle.superClass_.dispose_.call(thisField)();
     thisField.gauge_ = null;
-    if (thisField.clickWrapper_) {
-      Blockly.unbindEvent_(thisField.clickWrapper_);
+    if (thisField.mouseDownWrapper_) {
```

Comment:
This probably broke the Gesture code.  Testing on iOS is usually a fast way to see if Gestures are broken.